README.md

```diff
@@ -1,128 +1,207 @@
+* [Compatability](#compatability)
+* [Features](#features)
+* [Backspace Menu](#backspace-menu)
+* [Advanced Options](#advanced-options)
+* [Statistics](#statistics)
+* [Game Specific Features](#game-specific-features)
+* [Credits](#credits)
+* [FAQ](#faq)
+
+## <a name="downloading"></a> **Downloading**
+If you want to download the tool, you can download the [latest release](https://github.com/touhouworldcup/thprac/releases/latest) here and the [latest beta](https://nightly.link/touhouworldcup/thprac/workflows/main/master/thprac.zip) here.
+
+## <a name="usage"></a> **Usage**
+You can use this tool in a variety of ways; the main ways to do this are **placing the executable in the same folder as your game**, **running the executable after running your game**, or **using the thprac launcher**.
+
+### **Using the folder**
+**This method will NOT work with Steam games.** You can run the thprac tool by placing the `thprac.exe` file in the same folder as your game. This method will also automatically detect vpatch. You must ensure that the game is updated to the latest version and follows one of the following naming schemes. This method also works with vpatch.
+- thXX.exe (most games)
+- 東方紅魔郷.exe (Embodiment of Scarlet Devil)
+- alcostg.exe (Uwabami Breakers)
+
+### **Running the Executable**
+**This method will work for most methods.** First, run the game in whichever method you want (whether it's thcrap, using vpatch, Steam, etc.) and then run `thcrap.exe`. thcrap will then detect the running game and attempt to attach itself to it. You may have to return to the menu screen to see thprac take effect.
+
+### **Using the Launcher**
+This video explains how to use the launcher.
 [![A quick and dirty overview of thprac 2.0](https://i.ytimg.com/vi/sRV4PDQceYo/maxresdefault.jpg)](https://www.youtube.com/watch?v=sRV4PDQceYo)
 
-## Compatibility
-thprac officially supports every version of Windows starting with **Windows Vista**. Windows XP may work using [One-Core-API](https://github.com/Skulltrail192/One-Core-API-Binaries), but this is NOT being actively tested.
+## <a name="compatability"></a> **Compatability**
+thprac is compatible with all Windows versions proceeding and including Windows Vista. You may attempt to apply thprac on Windows XP using the [One-Core-API](https://github.com/Skulltrail192/One-Core-API-Binaries) but this is not guaranteed to work.
 
-**Wine** and the Steam Deck are also officially supported. Note that the backspace menu is also specifically coded to work well with the Steam Deck (not tested though cause I don't have a Steam Deck)
+thprac is also compatible with **Wine** and the Steam Deck. Do note that the backspace menu is tailored to work well with the Steam Deck, although this is untested.
+
+## <a name="features"></a> **Features**
+thprac features an enhanced practice mode that supports all main games along with Great Fairy Wars and Uwabami Breakers; do note that thprac **does not support Phantasmagoria of Flower View**.
```

Comment:
thprac does support PoFV (and both versions of UDoALG, though support for those games works differently). I consider TH09 Tools and TH19 Tools to basically be the enchanced practice mode for those games. Actually, looking one line further down, I understand why you wrote that. You wrote that because it says "Does not support PoFV" in the line you removed.